hlf/chaincode/operation.md

```diff
@@ -0,0 +1,187 @@
+# Hyperledger Operation Api
+
+In the following, the chaincode api for the "Operation"-Chaincode is described.
+This contains all Transactions and Models provided for this domain.
+The Errors returned are defined [here](errors.md#Errors).
+
+## Transactions
+
+### ApproveTransaction
+- ID = approveTransaction
+- Send
+    - contractName, transactionName, param1, ..., paramN :: String
+- Receive
+    - operationData :: [OperationData](#OperationData) 
+      -  Description: Success, returns the list of approvals.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes are not well formatted, they are listed in invalidParams.  
+       For detailed informations see [Input Checks](#Checks).
+
+### RejectTransaction
+- ID = rejectTransaction
+- Send
+    - operationId :: String
+    - rejectMessage :: String
+- Receive
+    - operationData :: [OperationData](#OperationData) 
+      -  Description: Success, returns the list of approvals.
+
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLUnprocessableLedgerState",
+        "title": "The state on the ledger does not conform to the specified format"
+      }
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the current model. This error should only occur if the model changes while the old ledger state remains without modification.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes are not well formatted, they are listed in invalidParams.  
+       For detailed informations see [Input Checks](#Checks).
+
+
+### GetOperationData
+- ID = getOperationData
+- Send
+    - operationId :: String
+- Receive
+    - operationData :: [OperationData](#OperationData)
+

```

Comment:
```suggestion

- [GenericError](errors.md#GenericError) 
      ```json
      {
        "type": "HLNotFound",
        "title": "There is no Operation for the given operationId"
      }
      ```
      - Description: This error is returned, if there is no operation for the specified operationId present on the ledger.
```